predictions_FB.py

```diff
@@ -0,0 +1,92 @@
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+from fbprophet import Prophet
+import yfinance as yf
+import math
+import os
+import sys
+import pandas as pd
+pd.options.mode.chained_assignment = None
+
+
+class suppress_stdout_stderr(object):
+    '''
+    Used to supress the verbose output of the inner workings of the FB prophet model
+    '''
+    def __init__(self):
+        self.null_fds = [os.open(os.devnull, os.O_RDWR) for x in range(2)]
+        self.save_fds = (os.dup(1), os.dup(2))
+
+    def __enter__(self):
+        os.dup2(self.null_fds[0], 1)
+        os.dup2(self.null_fds[1], 2)
+
+    def __exit__(self, *_):
+        os.dup2(self.save_fds[0], 1)
+        os.dup2(self.save_fds[1], 2)
+        os.close(self.null_fds[0])
+        os.close(self.null_fds[1])
+
+def fb_predict(dfi, plot = False, days =1):
```

Comment:
Can we add type hints to each parameter to help the calls understand what kind of arguments are to be sent?

These are typically defined using the `:` operator and you can even assign a default value using `=` in combination with this.

So for this, I imagine it would be somewhat like:
```Python
def fb_predict(dfi: file, plot: bool=False, days: int=1, verbosity: bool=True):
```